cmd/query/app/static_handler_test.go

```diff
@@ -51,3 +53,28 @@ func TestDefaultStaticAssetsRoot(t *testing.T) {
 	handler := NewStaticAssetsHandler("")
 	assert.Equal(t, "jaeger-ui-build/build/", handler.staticAssetsRoot)
 }
+
+func TestRegisterRoutesHandler(t *testing.T) {
+	r := mux.NewRouter()
+	handler := NewStaticAssetsHandler("fixture/")
+	handler.RegisterRoutes(r)
+	server := httptest.NewServer(r)
+	expectedRespString := "Test Favicon\n"
+	defer server.Close()
+
+	httpClient = &http.Client{
+		Timeout: 2 * time.Second,
+	}
+
+	url := fmt.Sprintf("%s/favicon.ico", server.URL)
+	resp, err := httpClient.Get(url)
```

Comment:
always check err first. `require.NoError(t, err)` before manipulating the response